### ISSUE TITLE
./btest 관련 질문드립니다

### ISSUE THREAD

**Issue by bellyHoney** [SNU-ARC/2023_fall_comarch_PA2#5]
안녕하세요,
제가 readme를 읽고 이해하기로 make btest를 하면 btest 파일이 만들어져야 하는데,
make btest를 실행하면 btest.c 16번 째 줄에 (#include <stdio.h>) 에러가 있다고 뜨면서 컴파일이 중단 되는데 (btest파일도 생성이 안 됩니다), 이거 어떻게 해결할 수 있나요?

감사합니다.

**Comment by bgsim** [comment]
안녕하세요. 조교 심봉근입니다.

혹시 PA2 ppt 5장에 있는
sudo apt-get install gcc gcc-multilib 을 하신 뒤에 하셨을까요?

시도해보시고 코멘트 부탁드립니다.

**Comment by bellyHoney** [comment]
네 다시 한번 sudo apt-get install gcc gcc-multilib을 했는데, 결과는 똑같습니다.
하나 특이점은 위 명령문을 돌렸을 때, 여러 에러가 뜨는데요 이러면 설치가 잘 안됐다는 뜻인가요?
사진으로 첨부해드렸습니다.

참고로 터미널에서 알려준 에러 해결법 apt-get update 그리고 apt-get --fix-missing을 해도 해결되지는 않았습니다.

감사합니다.
<img width="734" alt="스크린샷 2023-10-09 121351" src="https://github.com/SNU-ARC/2023_fall_comarch_PA2/assets/147307221/74deeec8-ac13-466e-9f1b-6b0bf65ce9b0">

**Comment by fyoon46** [comment]
저의 경우에는 sudo apt update 를 했더니 해결됐었습니다.

**Comment by bgsim** [comment]
정확한 원인은 알 수 없지만 네트워크 문제이거나 apt repository가 꼬인것 같습니다

우선 네트워크 환경을 먼저 확인하신 뒤에

sudo rm -rf /var/lib/apt/lists/* -vf
로 apt repository를 초기화시키고
apt-get update
apt-get upgrade 
를 통해 정상화 한 뒤에
apt-get install gcc gcc-multilib
를 재시도 해주시길 부탁드리겠습니다.

만약 이렇게해도 실패하신다면 환경설정에 큰 힘을 들이는것보다 pa1에서 제공해드렸던 vm을 활용해보시거나 docker를 제공해드리도록 하겠습니다.

부디 해결되길 바라겠습니다

**Comment by bellyHoney** [comment]
@fyoon46 @bgsim 도움 주셔서 너무나도 감사합니다
잘 참고해서 해결했습니다. 감사합니다.
